### DISCUSSION TITLE
how does yattee handle invidious 1080p?

### DISCUSSION THREAD

**Discussion by piuvas** [yattee/yattee#571] [Q&A]
1080 videos LOAD on yattee even without dash support. i was wondering how it's done since on invidious you need dash to load 1080 streams.

maybe they fallback to 720p and i just can't notice the difference?

**Comment by ghost** [comment]
I think it can also do 4k!
I don't have a 4k display so I'm not certain but when I set it to 4k, I can feel better bitrate of 4k and Yattee gets more buffering time,
so I don't think it is just falling back to 720p. I don't know what kind of sorcery they did but this is incredible!
Yattee was getting unusable because of most piped instance being shut down, broken, or slow.
This update solved that issue by supporting high res in private invidious instance!

**Reply by piuvas** [reply]
ikr? i think it's handling the video by itself whenever there's no proxy involved.
